packages/template/-private/dsl/invoke.d.ts

```diff
@@ -66,7 +66,7 @@ export declare function yieldToBlock<Context extends AnyContext, K extends keyof
  *     applySplattributes<typeof 𝚪.element, ElementForComponent<typeof AnotherComponent>>();
  */
 export declare function applySplattributes<
-  SourceElement extends Element,
+  SourceElement extends HTMLOrSVGElement,
```

Comment:
See below, but I think `Element` is correct here (`SVGOrHTMLElement` knows nothing about attributes)